vamb/__main__.py

```diff
@@ -1583,6 +1827,30 @@ def run_taxonomy_predictor(opt: TaxometerOptions):
     )
 
 
+def run_taxonomy_cross_validation(opt: TaxometerOptions):
+    composition, abundance = load_composition_and_abundance(
+        opt.general,
+        opt.composition,
+        opt.abundance,
+        vamb.vambtools.BinSplitter.inert_splitter(),
+    )
+    abundance, tnfs, lengths, _ = (
```

Comment:
Why assign the identifiers to `_`?

Reply:
as an indication that I don't use them later